tests/unit/cluster/cluster-reliable-meet.tcl

```diff
@@ -75,3 +78,162 @@ tags {tls:skip external:skip cluster} {
 
 set ::singledb $old_singledb
 
+proc cluster_get_first_node_in_handshake id {
```

Comment:
Would recommend to move away from all the length comparison being made to discover the state of a node. Either utilize `CLUSTER LINKS` or do string comparison on `CLUSTER NODES` output.